package.json

```diff
@@ -15,7 +15,8 @@
   "main": "dist/bundle.min.js",
   "types": "tools/terser.d.ts",
   "bin": {
-    "terser": "bin/uglifyjs"
+    "terser": "bin/terser",
+    "uglifyjs": "bin/uglifyjs"
```

Comment:
You can drop this line here, `terser` only ever had a single binary. Adding this line would make terser overwrite the uglifyjs binary with bin/uglifyjs. Keep the bin/uglifyjs file itself though, it will be useful for git bisecting.

Reply:
Yes, you're right. I totally missed that. But I don't see the usefulness of keeping ´bin/uglifyjs` 🤔